src/modules/common/components/ListItem.js

```diff
@@ -13,52 +19,42 @@ const StyledTouchableOpacity = styled.TouchableOpacity`
 `
 
 const Thumbnail = styled(FastImage)`
-  width: 100px;
-  height: 100px;
+  width: 75px;
+  height: 75px;
   flex-shrink: 0;
-  padding: 15px 5px;
-  object-fit: contain;
 `
 
 const Description = styled.View`
-  display: flex;
+  flex: 1;
   height: 100%;
-  min-width: 1px; /* needed to enable line breaks for too long words, exact value doesn't matter */
   flex-direction: column;
   flex-grow: 1;
-  padding: 15px 10px 0;
+  padding: 0 10px;
 `
 
 const Title = styled.Text`
   font-weight: 700;
 `
 
 type PropsType = {|
-  thumbnail?: string,
+  thumbnail?: {uri: string} | string,
```

Comment:
This can be renamed to `thumbnailUrl` with the type `string`

Reply:
It can be renamed, but the type has to stay like it is. We sometimes want to display a specific thumbnail from a link (hence `{uri: string}`) but if none is given display a Placeholder thumbnail from assets, which we have to pass as a string. See [here](https://github.com/Integreat/integreat-react-native-app/pull/15/files#diff-5f5945d35181867cf92ff917f4eb2e98R33).

Reply:
If would be interresting to know what value EventPlaceholder3 has here: import EventPlaceholder3 from '../assets/EventPlaceholder3.jpg'

Its probably already an URI or URL. So this type can probably be simplified.
Or use the type of the Image if you can find it somewhere: ImageSourcePropType https://facebook.github.io/react-native/docs/image#source

Reply:
EventPlaceholder has type `number`. As far as I can see there is no other way to do it.